test/conformance/api/v1alpha1/errorcondition_test.go

```diff
@@ -190,7 +190,7 @@ func TestContainerExitingMsg(t *testing.T) {
 						return true, nil
 					}
 					t.Logf("Reason: %s ; Message: %s ; Status: %s", cond.Reason, cond.Message, cond.Status)
-					return true, fmt.Errorf("The configuration %s was not marked with expected error condition (Reason=\"%s\", Message=\"%s\", Status=\"%s\"), but with (Reason=\"%s\", Message=\"%s\", Status=\"%s\")",
+					return true, fmt.Errorf("The configuration %s was not marked with expected error condition (Reason=%q, Message=%q, Status=%q), but with (Reason=%q, Message=%q, Status=%q)",
```

Comment:
ok I'll just now ask for all of them :- no more commenting on each one

Reply:
Markus is not making changes, just changing `\"%s\"` to `%q`. What you are asking for dug is real change.